source/renderer/app/components/wallet/settings/ChangeSpendingPasswordDialog.js

```diff
@@ -255,7 +279,7 @@ export default class ChangeSpendingPasswordDialog extends Component<Props> {
                 this.handleDataChange('currentPasswordValue', value)
               }
               {...currentPasswordField.bind()}
-              error={currentPasswordField.error}
+              error={currentPasswordField.error || currentPasswordError}
```

Comment:
@DeeJayElly remove `currentPasswordError` because that error will occur below inputs if API returns that current password is not correct

Reply:
Removed